src/LoginFormContainer.test.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import LoginFormContainer from './LoginFormContainer';
+
+jest.mock('react-redux');
+
+describe('LoginFormContainer', () => {
+  const dispatch = jest.fn();
+
+  beforeEach(() => {
+    dispatch.mockClear();
+
+    useDispatch.mockImplementation(() => dispatch);
+
+    useSelector.mockImplementation((selector) => selector({
+      loginFields: {
+        email: 'test@test',
+        password: '1234',
+      },
+    }));
+  });
+
+  const renderLoginFormContainer = () => render(
+    <LoginFormContainer />,
+  );
+
+  context('when a form field is changed', () => {
+    it('occurs changeLoginField', () => {
+      const { getByLabelText } = renderLoginFormContainer();
+
+      const input = getByLabelText('E-mail');
+
+      const value = 'newemail@email.com';
+
+      fireEvent.change(input, {
+        target: { value },
+      });
+
+      expect(dispatch).toBeCalledTimes(1);
```

Comment:
```suggestion
      expect(dispatch).toBeCalledWith({
        type: 'changeLoginField',
        payload: { name: 'email', value: 'new email' },
      });
```

좀 더 구체적인 테스트가 가능할 것 같네요.